conf/genotoul.config

```
@@ -0,0 +1,27 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'The Genotoul cluster profile'
+  config_profile_contact = 'support.bioinfo.genotoul@inra.fr'
+  config_profile_url = 'http://bioinfo.genotoul.fr/'
+}
+
+singularity {
+  // need one image per execution
+  enabled = true
+  runOptions = '-B /bank -B /work2 -B /work -B /save -B /home'
+  
+}
+
+process {
+  executor = 'slurm'
+}
+
+params {
+  saveReference = true
+  igenomesIgnore = true
```

could you also add `igenomes_ignore = true` here please?
we're transitioning to snake_case parameter names and should keep both for now. see https://github.com/nf-core/configs/pull/77 for reference.

same applies for `saveReference`

---

You know you can make suggestions?
```suggestion
  igenomes_ignore = true
```

---

> You know you can make suggestions?

I think this only works for you @nf-core/core people or if my review was requested explicitly

---

By the way I think both `igenomeIgnore` and `igenomesIgnore` should be here for now, because not all pipelines have been transitioned yet

---

you mean `igenomesIgnore` and `igenomes_ignore`?

---

for the record: I still think this will cause problems for some pipelines